### PR TITLE
Upload header templates

### PR DESCRIPTION
**Pull request by Nilrre**

### CODE REVIEW

**Review by BorisHo1 — APPROVED** [review]

**Review by CMoore455 — APPROVED** [review]

**Comment by Tyeboe** [comment]
Don't make it an entire page. You're just pushing up samples for a header that would be contained inside the body tag of another document.

**Review by Tyeboe — CHANGES_REQUESTED** [review]
^
  |

File: c.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div class="navbar">
+    <div>Link</label>
```
Comment (Tyeboe) [inline c.html]: Opening divs and closing labels

File: d.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div class="navbar">
+    <div>Link</label>
+    <div>Link2</label>
```
Comment (Tyeboe) [inline d.html]: Opening divs and closing labels

File: e.html [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<div class="navbar">
+    <div>Link</label>
+    <div>Link2</label>
```
Comment (Tyeboe) [inline e.html]: Opening divs and closing labels

**Review by Tyeboe — CHANGES_REQUESTED** [review]

**Review by Tyeboe — APPROVED** [review]

**Pull request merged**